djnro/settings.py

```diff
@@ -77,28 +77,33 @@
 #   'django.contrib.staticfiles.finders.DefaultStorageFinder',
 )
 
-# List of callables that know how to import templates from various sources.
-TEMPLATE_LOADERS = (
-    'django.template.loaders.filesystem.Loader',
-    'django.template.loaders.app_directories.Loader',
-#   'django.template.loaders.eggs.Loader',
-)
-
-TEMPLATE_CONTEXT_PROCESSORS = (
-    # Required so that RequestContext is passed into
-    # template
-    'django.contrib.auth.context_processors.auth',
-    'django.template.context_processors.debug',
-    'django.template.context_processors.i18n',
-    'django.template.context_processors.media',
-    'django.template.context_processors.request',
-    'edumanage.context_processors.country_code',
-    'edumanage.context_processors.cat_instances',
-    'edumanage.context_processors.manage_login_methods',
-    'edumanage.context_processors.reverse_urls',
-    'social.apps.django_app.context_processors.backends',
-    'social.apps.django_app.context_processors.login_redirect',
-)
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [
+            os.path.join(PROJECT_DIR, 'templates/'),
+            os.path.join(BASE_DIR, 'templates/'),
+        ],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                # Required so that RequestContext is passed into
+                # template
+                'django.contrib.auth.context_processors.auth',
+                'django.template.context_processors.debug',
+                'django.template.context_processors.i18n',
+                'django.template.context_processors.media',
+                'django.template.context_processors.request',
+                'edumanage.context_processors.country_code',
+                'edumanage.context_processors.cat_instances',
+                'edumanage.context_processors.manage_login_methods',
+                'edumanage.context_processors.reverse_urls',
+                'social.apps.django_app.context_processors.backends',
+                'social.apps.django_app.context_processors.login_redirect',
+            ],
+        },
+    },
```

Comment:
Trailing comma's could be removed - they are only necessary for single element tuples to be interpreted correctly.

Reply:
Agreed with @cangus to keep the trailing comma to avoid spurious (git) changes on the last line when adding another line - and to stay consistent with the rest of the code.

Reply:
Trailing commads are a good thing in this case, per PEP8.